### PR TITLE
chore(doc): add archlinux installation method

### LINKED ISSUE #61: Versioning?
Hi there,

I would like to create an aur package (arch linux users repo) for `try` but I saw that there is no version at all or any git tag.

Do you have in plan to release any version soon?

thanks

ps may I also suggest [bats](https://github.com/bats-core/bats-core) for the testing part?

### PR DESCRIPTION
**Pull request by brokenpip3**
fix #61

### CODE REVIEW

**Review by ericzty — APPROVED** [review]
lg2m

File: README.md [hunk 1]
```diff
@@ -41,6 +41,22 @@ You only need the [`try` script](https://raw.githubusercontent.com/binpash/try/m
 $ git clone https://github.com/binpash/try.git
 ```
 
+#### Archlinux
+
+`Try` is present in aur, you can install it with your preferred aur helper:
```
Comment (mgree) [inline README.md:46]: nits: I think it should be "Arch Linux" and "AUR" to match what's on their websites.

**Review by mgree — CHANGES_REQUESTED** [review]
Two tiny typographical nits. Thank you for doing this!

File: README.md (hunk 1 above)
Comment (brokenpip3) [reply README.md]: looks good now? :)

**Pull request merged**